hosted/eks/p1/p1_import_test.go

```diff
@@ -0,0 +1,208 @@
+package p1_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+
+	"github.com/rancher/hosted-providers-e2e/hosted/eks/helper"
+	"github.com/rancher/hosted-providers-e2e/hosted/helpers"
+)
+
+var _ = Describe("P1Import", func() {
+	var cluster *management.Cluster
+
+	Context("Importing/Editing a cluster with invalid config", func() {
```

Comment:
nit: Might be better to have a separate context inside `When("a cluster is imported",{})` for invalid update/editing configs, wdyt?

Reply:
Right, Before/After Each would also be re-used.